CMakeLists.txt

```diff
@@ -142,6 +143,11 @@ set(spotless_GIT_REPOSITORY https://github.com/RobotLocomotion/spotless-pod.git)
 set(spotless_GIT_TAG abb1f96947a244c069746c24de95edaeb84e7555)
 set(spotless_IS_CMAKE_POD TRUE)
 set(spotless_IS_PUBLIC TRUE)
+set(textbook_GIT_REPOSITORY https://github.com/rdeits/underactuated.git)
+set(textbook_GIT_TAG 95d64c30093e2f82c9194c71fc45a91d3c25d246)
```

Comment:
is this one case where we should consider referencing master directly?  otherwise it seems unlikely that we'll be updating it very often.  maybe just until the end of the term?